cmd/cloudFoundryCreateSpace.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/SAP/jenkins-library/pkg/cloudfoundry"
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+)
+
+func cloudFoundryCreateSpace(config cloudFoundryCreateSpaceOptions, telemetryData *telemetry.CustomData) {
+
+	c := command.Command{}
+	cf := cloudfoundry.CFUtils{Exec: &command.Command{}}
+
+	// reroute command output to logging framework
+	c.Stdout(log.Writer())
+	c.Stderr(log.Writer())
+
+	err := runCloudFoundryCreateSpace(&config, telemetryData, cf, &c)
+
+	if err != nil {
+		log.Entry().WithError(err).Fatal("step execution failed")
+	}
+
+}
+
+func runCloudFoundryCreateSpace(config *cloudFoundryCreateSpaceOptions, telemetryData *telemetry.CustomData, cf cloudfoundry.CFUtils, s command.ShellRunner) (err error) {
+
+	var c = cf.Exec
```

Comment:
Strange to re-use the ExecRunner from cf here. Technically it works, but would't it be clearer to hand-over that via signature (which might considered to be weird, too - since we have the same instance in `s` and in `c`) or cast `s` to `ExecRunner` since the same instance is actually both a `ShellRunner` and an `ExecRunner`. If that remains as it is I will approve nevertheless.

Reply:
I agree that this looks strange. Could you use your `ShellRunner` `s` instead of `c` below?

Reply:
I used shellRunner as we do not have an option to use cf exec as we wanted to run 2 commands right after one.  But, cf exec makes the things easier and I used it when we wanted to exec only cf based  single command. Is there any problem with using both as required?

Reply:
Not a problem directly. It is more about code elegance :-) maybe together with a grain of maintainability. The clearest approach IMO would be to hand over the instance with two variabls `s ShellRunner` and `c ExecRunner`. But in that case one might also ask: why handing it over twice ... I it not that easy. Again: for me no reason for not approving ...